src/ReactiveUI/Platforms/windows-common/RoutedViewHost.cs

```diff
@@ -55,6 +58,8 @@ class RoutedViewHost : TransitioningContentControl, IActivatableView, IEnableLog
         public static readonly DependencyProperty ViewContractObservableProperty =
             DependencyProperty.Register("ViewContractObservable", typeof(IObservable<string>), typeof(RoutedViewHost), new PropertyMetadata(Observable<string>.Default));
 
+        private string? _viewContract;
```

Comment:
Seems like you aren't setting this anywhere?

Reply:
Thanks, I will fix that before going to sleep.

Reply:
Updated